diff --git a/packages/platforms/accton/x86-64/asgvolt64/onlp/builds/Makefile b/packages/platforms/accton/x86-64/asgvolt64/onlp/builds/Makefile
index e7437cb2..78ef4fd8 100755
--- a/packages/platforms/accton/x86-64/asgvolt64/onlp/builds/Makefile
+++ b/packages/platforms/accton/x86-64/asgvolt64/onlp/builds/Makefile
@@ -1,2 +1,2 @@
-FILTER=src
+FILTER=x86_64_accton_asgvolt64
 include $(ONL)/make/subdirs.mk
