diff --git a/packages/platforms/skeleton/x86-64/skelmachine/onlp/builds/Makefile b/packages/platforms/skeleton/x86-64/skelmachine/onlp/builds/Makefile
index e7437cb2..78ef4fd8 100755
--- a/packages/platforms/skeleton/x86-64/skelmachine/onlp/builds/Makefile
+++ b/packages/platforms/skeleton/x86-64/skelmachine/onlp/builds/Makefile
@@ -1,2 +1,2 @@
-FILTER=src
+FILTER=x86_64_skeleton_skelmachine
 include $(ONL)/make/subdirs.mk
